test/module/irohad/ametsuchi/ametsuchi_fixture.hpp

```diff
@@ -78,6 +80,12 @@ namespace iroha {
 
       std::shared_ptr<soci::session> sql;
 
+      std::shared_ptr<shared_model::proto::ProtoCommonObjectsFactory<
+          shared_model::validation::FieldValidator>>
+      factory =
```

Comment:
Why not ```auto```?

Reply:
I think because it's declaration as well

Reply:
It is a class member